src/redux/reducers/adminReducers.js

```diff
@@ -0,0 +1,43 @@
+import { GET_ALL_USERS, GET_ALL_USERS_FAIL, DELETE_USER, DELETE_USER_FAIL, DELETED_ALL_SERVICES_OF_VENDOR } from '../actions/type';
+
+const initialState = {
+  users: [],
+  status: null,
+  deletedUserStatus: null,
+  deletedAllServicesOfVendorStatus: null,
+}
+
+export default function (state = initialState, action) {
+  switch (action.type) {
+    case GET_ALL_USERS:
+      return {
+        users: action.payload,
+        status: 200
+      }
+    case GET_ALL_USERS_FAIL:
+      return {
+        ...state,
+        status: null
+      }
+    case DELETE_USER:
+      return {
+        ...state,
+        status: 200,
+        deletedUserStatus: 1
```

Comment:
Don't create a state for this. Rather call the notification of antd after calling the api in .then() promise display successfully deleted if status 200 else error. Also no need of deletedAllServicesOfVendorStatus. Just check if the user is also a vendor and it to the notification message.

Reply:
@ashwaniYDV You have not resolved this issue. Also now when I delete a vendor the User deleted notification comes multiple times. Correct that.

Reply:
@harsh-jindal That issue of multiple messages is tackled by using the delete all service API.

Reply:
@harsh-jindal I've used status and statusType in the admin state to differentiate between which notification to show since all have status code 200